packages/core/src/FileUploader/FileList/FileList.js

```diff
@@ -1,20 +1,22 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { withStyles } from "@material-ui/core";
-import { setUid } from "../..";
+import { setId, useUniqueId } from "../..";
 import File from "../File";
 import styles from "./styles";
 
 const FileList = ({ id, classes, list = [], removeFileButtonLabel, onFileRemoved }) => {
+  const elementId = useUniqueId(id, "hvfilelist");
+
   const hasFiles = list.length > 0;
   if (!hasFiles) return null;
 
   return (
-    <ul id={id} className={classes.list}>
+    <ul id={setId(id, "list")} className={classes.list}>
       {list.map((data) => (
         <File
           key={data.id}
-          id={setUid(id, data.id)}
+          id={setId(elementId, "values")}
```

Comment:
Wasn't this meant to be passed to the `key` instead? Would need the `index` though.

If this was intended then:
- All the `File`s will be getting the same `id`, which is wrong
- We're adding extra stuff to the DOM that is unnecessary and unasked for
  - I don't like polluting the DOM with useless `id`s. Also, someone might use them for testing, which is bad

^ The snapshot tests actually caught these two _issues_, but nobody reads snapshots 🤷